packages/main/src/pages/protected/workspace/Workspace.jsx

```diff
@@ -84,38 +91,29 @@ export default function Index() {
         </>
 
         <WorkspaceWrapperStyle>
-          <div id="single-spa-loader">
-            <WorkspaceLoading text="Initializing Workspace..." />
-          </div>
-          <div id="zuri-plugin-load-section"></div>
-          <Switch>
-            <Route exact path="/workspace/:workspaceId">
-              <h1>Welcome to your Workspace</h1>
-            </Route>
-            <Route path="/workspace/:workspaceId/LiveBroadcast">
-              <LiveBroadcast />
-            </Route>
+          <div id="zuri-plugin-load-section">
+            <Switch>
```

Comment:
You omitted some very important code up there. Please review and return that loader and the div that you removed

Reply:
So for this, I wrapped the switches inside the zuri plugin load section.

Reply:
she's referring to the single spa loader